CRITICAL - 2023-11-03 01:19:20 --> Undefined variable $ruta_carpeta
#0 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(3328): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 3328)
#1 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->abrir_carpeta_archivos()
#2 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2023-11-03 01:20:06 --> Undefined variable $ruta_carpeta
#0 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(3328): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 3328)
#1 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->abrir_carpeta_archivos()
#2 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2023-11-03 01:20:12 --> Undefined variable $ruta_carpeta
#0 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(3328): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 3328)
#1 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->abrir_carpeta_archivos()
#2 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2023-11-03 01:55:34 --> Call to undefined method App\Controllers\Inventarios::Image()
#0 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarResultadoDeAnalisis()
#1 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#2 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2023-11-03 02:00:15 --> FPDF error: Not a JPEG file: http://etic-system.local/img/sistema/iso_img.jpg
#0 C:\xampp\htdocs\etic-system\app\ThirdParty\fpdf\FPDF.php(1270): App\ThirdParty\fpdf\FPDF->Error('Not a JPEG file...')
#1 C:\xampp\htdocs\etic-system\app\ThirdParty\fpdf\FPDF.php(913): App\ThirdParty\fpdf\FPDF->_parsejpg('http://etic-sys...')
#2 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(2484): App\ThirdParty\fpdf\FPDF->Image('http://etic-sys...', 30, 30)
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarResultadoDeAnalisis()
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-11-03 02:01:27 --> FPDF error: Not a JPEG file: http://etic-system.local/img/sistema/iso_img.jpg
#0 C:\xampp\htdocs\etic-system\app\ThirdParty\fpdf\FPDF.php(1270): App\ThirdParty\fpdf\FPDF->Error('Not a JPEG file...')
#1 C:\xampp\htdocs\etic-system\app\ThirdParty\fpdf\FPDF.php(913): App\ThirdParty\fpdf\FPDF->_parsejpg('http://etic-sys...')
#2 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(2484): App\ThirdParty\fpdf\FPDF->Image('http://etic-sys...', 30, 30, 10)
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarResultadoDeAnalisis()
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-11-03 02:04:14 --> fopen(http://etic-system.local/img/sistema/iso_img.png): Failed to open stream: HTTP request failed! HTTP/1.1 404 Not Found

#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'fopen(http://et...', 'C:\\xampp\\htdocs...', 1285)
#1 C:\xampp\htdocs\etic-system\app\ThirdParty\fpdf\FPDF.php(1285): fopen('http://etic-sys...', 'rb')
#2 C:\xampp\htdocs\etic-system\app\ThirdParty\fpdf\FPDF.php(913): App\ThirdParty\fpdf\FPDF->_parsepng('http://etic-sys...')
#3 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(2484): App\ThirdParty\fpdf\FPDF->Image('http://etic-sys...', 30, 30, 10)
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarResultadoDeAnalisis()
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#6 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2023-11-03 10:28:39 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 10:28:39 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('I1SZWF4E-8KXT-G...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('I1SZWF4E-8KXT-G...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 10:28:39 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 10:28:39 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('I1SZWF4E-8KXT-G...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('I1SZWF4E-8KXT-G...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 10:38:06 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 10:38:06 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('I1SZWF4E-8KXT-G...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('I1SZWF4E-8KXT-G...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 10:38:07 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 10:38:07 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('I1SZWF4E-8KXT-G...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('I1SZWF4E-8KXT-G...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 10:41:51 --> Unknown column 'Id_Cliente' in 'field list'
CRITICAL - 2023-11-03 10:41:51 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\UsuariosMdl.php(64): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Login.php(33): App\Models\UsuariosMdl->obtenerUsuario(Array)
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Login->acceso()
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 10:49:24 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 10:49:24 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\InspeccionesMdl.php(99): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(46): App\Models\InspeccionesMdl->datos_inspeccion_restaurar()
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->index()
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-11-03 10:58:45 --> Undefined array key 0
#0 C:\xampp\htdocs\etic-system\app\Controllers\Inspecciones.php(304): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 304)
#1 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inspecciones->cargar_bd_inspeccion()
#2 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inspecciones))
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2023-11-03 11:12:53 --> opendir(C:\xampp\htdocs\etic-system\public/Archivos_ETIC/inspecciones/1924/Imagenes): El sistema no puede encontrar la ruta especifica (code: 3)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'opendir(C:\\xamp...', 'C:\\xampp\\htdocs...', 987)
#1 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(987): opendir('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->explorarArchivos()
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
ERROR - 2023-11-03 11:12:53 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:12:53 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 11:12:54 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:12:54 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-11-03 11:13:30 --> opendir(C:\xampp\htdocs\etic-system\public/Archivos_ETIC/inspecciones/1924/Imagenes): El sistema no puede encontrar la ruta especifica (code: 3)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'opendir(C:\\xamp...', 'C:\\xampp\\htdocs...', 987)
#1 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(987): opendir('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->explorarArchivos()
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
ERROR - 2023-11-03 11:13:31 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:13:31 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 11:13:32 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:13:32 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-11-03 11:13:43 --> Too few arguments to function App\Controllers\Inspecciones::descargar_bd_exportar(), 0 passed in C:\xampp\htdocs\etic-system\system\CodeIgniter.php on line 824 and exactly 1 expected
#0 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inspecciones->descargar_bd_exportar()
#1 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inspecciones))
#2 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2023-11-03 11:14:04 --> opendir(C:\xampp\htdocs\etic-system\public/Archivos_ETIC/inspecciones/1924/Imagenes): El sistema no puede encontrar la ruta especifica (code: 3)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'opendir(C:\\xamp...', 'C:\\xampp\\htdocs...', 987)
#1 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(987): opendir('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->explorarArchivos()
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
ERROR - 2023-11-03 11:14:04 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:14:04 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 11:14:04 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:14:04 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-11-03 11:14:19 --> opendir(C:\xampp\htdocs\etic-system\public/Archivos_ETIC/inspecciones/1924/Imagenes): El sistema no puede encontrar la ruta especifica (code: 3)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'opendir(C:\\xamp...', 'C:\\xampp\\htdocs...', 987)
#1 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(987): opendir('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->explorarArchivos()
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
ERROR - 2023-11-03 11:14:20 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:14:20 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 11:14:21 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:14:21 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-11-03 11:14:33 --> Too few arguments to function App\Controllers\Inspecciones::descargar_bd_exportar(), 0 passed in C:\xampp\htdocs\etic-system\system\CodeIgniter.php on line 824 and exactly 1 expected
#0 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inspecciones->descargar_bd_exportar()
#1 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inspecciones))
#2 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2023-11-03 11:19:49 --> Maximum execution time of 120 seconds exceeded
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-11-03 11:24:40 --> Maximum execution time of 120 seconds exceeded
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-11-03 11:30:12 --> Maximum execution time of 120 seconds exceeded
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-11-03 11:34:41 --> opendir(C:\xampp\htdocs\etic-system\public/Archivos_ETIC/inspecciones/1924/Imagenes): El sistema no puede encontrar la ruta especifica (code: 3)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'opendir(C:\\xamp...', 'C:\\xampp\\htdocs...', 987)
#1 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(987): opendir('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->explorarArchivos()
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
ERROR - 2023-11-03 11:34:42 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:34:42 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 11:34:43 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:34:43 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-11-03 11:35:05 --> Too few arguments to function App\Controllers\Inspecciones::descargar_bd_exportar(), 0 passed in C:\xampp\htdocs\etic-system\system\CodeIgniter.php on line 824 and exactly 1 expected
#0 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inspecciones->descargar_bd_exportar()
#1 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inspecciones))
#2 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2023-11-03 11:39:31 --> opendir(C:\xampp\htdocs\etic-system\public/Archivos_ETIC/inspecciones/1924/Imagenes): El sistema no puede encontrar la ruta especifica (code: 3)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'opendir(C:\\xamp...', 'C:\\xampp\\htdocs...', 987)
#1 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(987): opendir('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->explorarArchivos()
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
ERROR - 2023-11-03 11:39:31 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:39:31 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 11:39:32 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:39:32 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-11-03 11:39:55 --> Too few arguments to function App\Controllers\Inspecciones::descargar_bd_exportar(), 0 passed in C:\xampp\htdocs\etic-system\system\CodeIgniter.php on line 824 and exactly 1 expected
#0 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inspecciones->descargar_bd_exportar()
#1 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inspecciones))
#2 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2023-11-03 11:44:22 --> opendir(C:\xampp\htdocs\etic-system\public/Archivos_ETIC/inspecciones/1924/Imagenes): El sistema no puede encontrar la ruta especifica (code: 3)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'opendir(C:\\xamp...', 'C:\\xampp\\htdocs...', 987)
#1 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(987): opendir('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->explorarArchivos()
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
ERROR - 2023-11-03 11:44:22 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:44:22 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 11:44:22 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:44:22 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-11-03 11:44:37 --> File not found: C:\xampp\htdocs\etic-system\public/Archivos_ETIC/bd_exportar_inspecciones/INSPECCION_NN
#0 C:\xampp\htdocs\etic-system\system\Files\File.php(46): CodeIgniter\Files\Exceptions\FileNotFoundException::forFileNotFound('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\etic-system\system\HTTP\DownloadResponse.php(107): CodeIgniter\Files\File->__construct('C:\\xampp\\htdocs...', true)
#2 C:\xampp\htdocs\etic-system\system\HTTP\ResponseTrait.php(723): CodeIgniter\HTTP\DownloadResponse->setFilePath('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\etic-system\app\Controllers\Inspecciones.php(284): CodeIgniter\HTTP\Response->download('INSPECCION_NN', NULL)
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inspecciones->descargar_bd_exportar()
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inspecciones))
#6 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-11-03 11:52:35 --> opendir(C:\xampp\htdocs\etic-system\public/Archivos_ETIC/inspecciones/1924/Imagenes): El sistema no puede encontrar la ruta especifica (code: 3)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'opendir(C:\\xamp...', 'C:\\xampp\\htdocs...', 987)
#1 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(987): opendir('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->explorarArchivos()
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
ERROR - 2023-11-03 11:52:36 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:52:36 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 11:52:37 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:52:37 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-11-03 11:52:49 --> File not found: C:\xampp\htdocs\etic-system\public/Archivos_ETIC/bd_exportar_inspecciones/INSPECCION_NN.sql
#0 C:\xampp\htdocs\etic-system\system\Files\File.php(46): CodeIgniter\Files\Exceptions\FileNotFoundException::forFileNotFound('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\etic-system\system\HTTP\DownloadResponse.php(107): CodeIgniter\Files\File->__construct('C:\\xampp\\htdocs...', true)
#2 C:\xampp\htdocs\etic-system\system\HTTP\ResponseTrait.php(723): CodeIgniter\HTTP\DownloadResponse->setFilePath('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\etic-system\app\Controllers\Inspecciones.php(284): CodeIgniter\HTTP\Response->download('INSPECCION_NN.s...', NULL)
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inspecciones->descargar_bd_exportar()
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inspecciones))
#6 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-11-03 11:57:03 --> opendir(C:\xampp\htdocs\etic-system\public/Archivos_ETIC/inspecciones/1924/Imagenes): El sistema no puede encontrar la ruta especifica (code: 3)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'opendir(C:\\xamp...', 'C:\\xampp\\htdocs...', 987)
#1 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(987): opendir('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->explorarArchivos()
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
ERROR - 2023-11-03 11:57:04 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:57:04 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 11:57:04 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:57:04 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-11-03 11:57:18 --> opendir(C:\xampp\htdocs\etic-system\public/Archivos_ETIC/inspecciones/1924/Imagenes): El sistema no puede encontrar la ruta especifica (code: 3)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'opendir(C:\\xamp...', 'C:\\xampp\\htdocs...', 987)
#1 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(987): opendir('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->explorarArchivos()
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
ERROR - 2023-11-03 11:57:18 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:57:18 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 11:57:18 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:57:18 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-11-03 11:57:31 --> File not found: C:\xampp\htdocs\etic-system\public/Archivos_ETIC/bd_exportar_inspecciones/INSPECCION_NN.sql
#0 C:\xampp\htdocs\etic-system\system\Files\File.php(46): CodeIgniter\Files\Exceptions\FileNotFoundException::forFileNotFound('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\etic-system\system\HTTP\DownloadResponse.php(107): CodeIgniter\Files\File->__construct('C:\\xampp\\htdocs...', true)
#2 C:\xampp\htdocs\etic-system\system\HTTP\ResponseTrait.php(723): CodeIgniter\HTTP\DownloadResponse->setFilePath('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\etic-system\app\Controllers\Inspecciones.php(284): CodeIgniter\HTTP\Response->download('INSPECCION_NN.s...', NULL)
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inspecciones->descargar_bd_exportar()
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inspecciones))
#6 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-11-03 12:00:48 --> opendir(C:\xampp\htdocs\etic-system\public/Archivos_ETIC/inspecciones/1924/Imagenes): El sistema no puede encontrar la ruta especifica (code: 3)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'opendir(C:\\xamp...', 'C:\\xampp\\htdocs...', 987)
#1 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(987): opendir('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->explorarArchivos()
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
ERROR - 2023-11-03 12:00:48 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 12:00:48 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 12:00:49 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 12:00:49 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-11-03 12:01:49 --> opendir(C:\xampp\htdocs\etic-system\public/Archivos_ETIC/inspecciones/1924/Imagenes): El sistema no puede encontrar la ruta especifica (code: 3)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'opendir(C:\\xamp...', 'C:\\xampp\\htdocs...', 987)
#1 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(987): opendir('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->explorarArchivos()
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
ERROR - 2023-11-03 12:01:49 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 12:01:49 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 12:01:50 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 12:01:50 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-11-03 12:03:33 --> opendir(C:\xampp\htdocs\etic-system\public/Archivos_ETIC/inspecciones/1924/Imagenes): El sistema no puede encontrar la ruta especifica (code: 3)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'opendir(C:\\xamp...', 'C:\\xampp\\htdocs...', 987)
#1 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(987): opendir('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->explorarArchivos()
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
ERROR - 2023-11-03 12:03:33 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 12:03:33 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 12:03:34 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 12:03:34 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-11-03 12:04:12 --> opendir(C:\xampp\htdocs\etic-system\public/Archivos_ETIC/inspecciones/1924/Imagenes): El sistema no puede encontrar la ruta especifica (code: 3)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'opendir(C:\\xamp...', 'C:\\xampp\\htdocs...', 987)
#1 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(987): opendir('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->explorarArchivos()
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
ERROR - 2023-11-03 12:04:13 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 12:04:13 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 12:04:13 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 12:04:13 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-11-03 12:04:34 --> opendir(C:\xampp\htdocs\etic-system\public/Archivos_ETIC/inspecciones/1924/Imagenes): El sistema no puede encontrar la ruta especifica (code: 3)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'opendir(C:\\xamp...', 'C:\\xampp\\htdocs...', 987)
#1 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(987): opendir('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->explorarArchivos()
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
ERROR - 2023-11-03 12:04:35 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 12:04:35 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 12:04:36 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 12:04:36 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 12:05:13 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 12:05:13 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 12:05:13 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 12:05:13 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 12:06:11 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 12:06:11 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 12:06:11 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 12:06:11 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('7B56774F-D176-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('7B56774F-D176-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-11-03 12:09:24 --> opendir(C:\xampp\htdocs\etic-system\public/Archivos_ETIC/inspecciones/1825/Imagenes): El sistema no puede encontrar la ruta especifica (code: 3)
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'opendir(C:\\xamp...', 'C:\\xampp\\htdocs...', 987)
#1 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(987): opendir('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->explorarArchivos()
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
ERROR - 2023-11-03 12:09:25 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 12:09:25 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('33441751-E75D-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('33441751-E75D-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 12:09:25 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 12:09:25 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('33441751-E75D-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('33441751-E75D-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 12:09:44 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 12:09:44 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('33441751-E75D-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('33441751-E75D-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 12:09:44 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 12:09:44 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Sitios.php(28): App\Models\SitiosMdl->obtenerRegistros('33441751-E75D-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Sitios->contactos_sitios('33441751-E75D-4...')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Sitios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-11-03 12:27:45 --> file_put_contents(C:/xampp/htdocs/etic-system/public/Archivos_ETIC/inspecciones/1825/Reportes/ETIC_BASELINE_INSPECCION_1825.pdf): Failed to open stream: No such file or directory
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_put_conten...', 'C:\\xampp\\htdocs...', 1049)
#1 C:\xampp\htdocs\etic-system\app\ThirdParty\fpdf\FPDF.php(1049): file_put_contents('C:/xampp/htdocs...', '%PDF-1.3\n3 0 ob...')
#2 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(1758): App\ThirdParty\fpdf\FPDF->Output('F', 'C:/xampp/htdocs...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarReporteBaseLine()
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-11-03 12:28:23 --> file_put_contents(C:/xampp/htdocs/etic-system/public/Archivos_ETIC/inspecciones/1825/Reportes/ETIC_BASELINE_INSPECCION_1825.pdf): Failed to open stream: No such file or directory
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_put_conten...', 'C:\\xampp\\htdocs...', 1049)
#1 C:\xampp\htdocs\etic-system\app\ThirdParty\fpdf\FPDF.php(1049): file_put_contents('C:/xampp/htdocs...', '%PDF-1.3\n3 0 ob...')
#2 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(1758): App\ThirdParty\fpdf\FPDF->Output('F', 'C:/xampp/htdocs...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarReporteBaseLine()
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-11-03 12:28:28 --> file_put_contents(C:/xampp/htdocs/etic-system/public/Archivos_ETIC/inspecciones/1825/Reportes/ETIC_PROBLEMAS_INSPECCION_1825.pdf): Failed to open stream: No such file or directory
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_put_conten...', 'C:\\xampp\\htdocs...', 1049)
#1 C:\xampp\htdocs\etic-system\app\ThirdParty\fpdf\FPDF.php(1049): file_put_contents('C:/xampp/htdocs...', '%PDF-1.3\n3 0 ob...')
#2 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(1535): App\ThirdParty\fpdf\FPDF->Output('F', 'C:/xampp/htdocs...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarReporteProblemas()
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-11-03 12:28:58 --> file_put_contents(C:/xampp/htdocs/etic-system/public/Archivos_ETIC/inspecciones/1825/Reportes/ETIC_BASELINE_INSPECCION_1825.pdf): Failed to open stream: No such file or directory
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_put_conten...', 'C:\\xampp\\htdocs...', 1049)
#1 C:\xampp\htdocs\etic-system\app\ThirdParty\fpdf\FPDF.php(1049): file_put_contents('C:/xampp/htdocs...', '%PDF-1.3\n3 0 ob...')
#2 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(1758): App\ThirdParty\fpdf\FPDF->Output('F', 'C:/xampp/htdocs...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarReporteBaseLine()
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-11-03 12:31:08 --> file_put_contents(C:/xampp/htdocs/etic-system/public/Archivos_ETIC/inspecciones/1825/Reportes/ETIC_BASELINE_INSPECCION_1825.pdf): Failed to open stream: No such file or directory
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'file_put_conten...', 'C:\\xampp\\htdocs...', 1049)
#1 C:\xampp\htdocs\etic-system\app\ThirdParty\fpdf\FPDF.php(1049): file_put_contents('C:/xampp/htdocs...', '%PDF-1.3\n3 0 ob...')
#2 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(1758): App\ThirdParty\fpdf\FPDF->Output('F', 'C:/xampp/htdocs...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarReporteBaseLine()
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 11:32:22 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:32:22 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(3192): App\Models\SitiosMdl->obtenerRegistros('33441751-E75D-4...')
#3 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(2203): App\Controllers\Inventarios->generarReporteListaProblemasExcel('2023-11-03', '2023-11-03')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarResultadoDeAnalisis()
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#6 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2023-11-03 11:32:34 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:32:34 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(3192): App\Models\SitiosMdl->obtenerRegistros('33441751-E75D-4...')
#3 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(2203): App\Controllers\Inventarios->generarReporteListaProblemasExcel('2023-11-03', '2023-11-03')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarResultadoDeAnalisis()
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#6 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2023-11-03 11:33:34 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:33:34 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(3192): App\Models\SitiosMdl->obtenerRegistros('33441751-E75D-4...')
#3 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(2203): App\Controllers\Inventarios->generarReporteListaProblemasExcel('2023-11-03', '2023-11-03')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarResultadoDeAnalisis()
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#6 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2023-11-03 11:34:10 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:34:10 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(3192): App\Models\SitiosMdl->obtenerRegistros('33441751-E75D-4...')
#3 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(2203): App\Controllers\Inventarios->generarReporteListaProblemasExcel('2023-11-03', '2023-11-03')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarResultadoDeAnalisis()
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#6 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2023-11-03 11:35:36 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:35:36 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(3192): App\Models\SitiosMdl->obtenerRegistros('33441751-E75D-4...')
#3 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(2203): App\Controllers\Inventarios->generarReporteListaProblemasExcel('2023-11-03', '2023-11-03')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarResultadoDeAnalisis()
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#6 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2023-11-03 11:35:59 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:35:59 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(3192): App\Models\SitiosMdl->obtenerRegistros('33441751-E75D-4...')
#3 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(2203): App\Controllers\Inventarios->generarReporteListaProblemasExcel('2023-11-03', '2023-11-03')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarResultadoDeAnalisis()
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#6 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2023-11-03 11:36:49 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:36:49 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(3192): App\Models\SitiosMdl->obtenerRegistros('33441751-E75D-4...')
#3 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(2203): App\Controllers\Inventarios->generarReporteListaProblemasExcel('2023-11-03', '2023-11-03')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarResultadoDeAnalisis()
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#6 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2023-11-03 11:37:34 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:37:34 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(3192): App\Models\SitiosMdl->obtenerRegistros('33441751-E75D-4...')
#3 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(2203): App\Controllers\Inventarios->generarReporteListaProblemasExcel('2023-11-03', '2023-11-03')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarResultadoDeAnalisis()
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#6 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2023-11-03 11:38:13 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:38:13 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(3192): App\Models\SitiosMdl->obtenerRegistros('33441751-E75D-4...')
#3 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(2203): App\Controllers\Inventarios->generarReporteListaProblemasExcel('2023-11-03', '2023-11-03')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarResultadoDeAnalisis()
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#6 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2023-11-03 11:38:32 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:38:32 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(2220): App\Models\SitiosMdl->obtenerRegistros('33441751-E75D-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarResultadoDeAnalisis()
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 11:38:50 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:38:50 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(3192): App\Models\SitiosMdl->obtenerRegistros('33441751-E75D-4...')
#3 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(2203): App\Controllers\Inventarios->generarReporteListaProblemasExcel('2023-11-03', '2023-11-03')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarResultadoDeAnalisis()
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#6 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2023-11-03 11:39:25 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:39:25 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(3192): App\Models\SitiosMdl->obtenerRegistros('33441751-E75D-4...')
#3 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(2203): App\Controllers\Inventarios->generarReporteListaProblemasExcel('2023-11-03', '2023-11-03')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarResultadoDeAnalisis()
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#6 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2023-11-03 11:39:43 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:39:43 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(3192): App\Models\SitiosMdl->obtenerRegistros('33441751-E75D-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarReporteListaProblemasExcel('2023-11-03', '2023-11-03')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 11:39:52 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:39:52 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(3192): App\Models\SitiosMdl->obtenerRegistros('33441751-E75D-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarReporteListaProblemasExcel('2023-11-03', '2023-11-03')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 11:40:30 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:40:30 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(3192): App\Models\SitiosMdl->obtenerRegistros('33441751-E75D-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarReporteListaProblemasExcel('2023-11-03', '2023-11-03')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2023-11-03 11:41:41 --> Unknown column 'Id_Grupo_Sitios' in 'field list'
CRITICAL - 2023-11-03 11:41:41 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\etic-system\app\Models\SitiosMdl.php(70): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(3192): App\Models\SitiosMdl->obtenerRegistros('33441751-E75D-4...')
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarReporteListaProblemasExcel('2023-11-03', '2023-11-03')
#4 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#5 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-11-03 13:02:25 --> ini_set(): Session ini settings cannot be changed after headers have already been sent
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'ini_set(): Sess...', 'C:\\xampp\\htdocs...', 71)
#1 C:\xampp\htdocs\etic-system\system\Session\Handlers\FileHandler.php(71): ini_set('session.save_pa...', 'C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\Config\Services.php(570): CodeIgniter\Session\Handlers\FileHandler->__construct(Object(Config\App), '127.0.0.1')
#3 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session(Object(Config\App), false)
#4 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#5 C:\xampp\htdocs\etic-system\system\Config\Services.php(563): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#6 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session()
#7 C:\xampp\htdocs\etic-system\system\Common.php(909): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#8 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(1545): session()
#9 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarReporteBaseLine()
#10 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#11 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-11-03 13:02:25 --> Uncaught ErrorException: Cannot modify header information - headers already sent by (output started at C:\xampp\htdocs\etic-system\app\Config\Events.php:32) in C:\xampp\htdocs\etic-system\system\Debug\Exceptions.php:107
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'Cannot modify h...', 'C:\\xampp\\htdocs...', 107)
#1 C:\xampp\htdocs\etic-system\system\Debug\Exceptions.php(107): header('HTTP/1.1 500 In...', true, 500)
#2 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#3 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-11-03 13:03:55 --> ini_set(): Session ini settings cannot be changed after headers have already been sent
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'ini_set(): Sess...', 'C:\\xampp\\htdocs...', 71)
#1 C:\xampp\htdocs\etic-system\system\Session\Handlers\FileHandler.php(71): ini_set('session.save_pa...', 'C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\Config\Services.php(570): CodeIgniter\Session\Handlers\FileHandler->__construct(Object(Config\App), '127.0.0.1')
#3 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session(Object(Config\App), false)
#4 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#5 C:\xampp\htdocs\etic-system\system\Config\Services.php(563): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#6 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session()
#7 C:\xampp\htdocs\etic-system\system\Common.php(909): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#8 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(1545): session()
#9 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarReporteBaseLine()
#10 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#11 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-11-03 13:03:55 --> Uncaught ErrorException: Cannot modify header information - headers already sent by (output started at C:\xampp\htdocs\etic-system\app\Config\Events.php:32) in C:\xampp\htdocs\etic-system\system\Debug\Exceptions.php:107
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'Cannot modify h...', 'C:\\xampp\\htdocs...', 107)
#1 C:\xampp\htdocs\etic-system\system\Debug\Exceptions.php(107): header('HTTP/1.1 500 In...', true, 500)
#2 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#3 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-11-03 13:06:14 --> ini_set(): Session ini settings cannot be changed after headers have already been sent
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'ini_set(): Sess...', 'C:\\xampp\\htdocs...', 71)
#1 C:\xampp\htdocs\etic-system\system\Session\Handlers\FileHandler.php(71): ini_set('session.save_pa...', 'C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\Config\Services.php(570): CodeIgniter\Session\Handlers\FileHandler->__construct(Object(Config\App), '127.0.0.1')
#3 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session(Object(Config\App), false)
#4 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#5 C:\xampp\htdocs\etic-system\system\Config\Services.php(563): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#6 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session()
#7 C:\xampp\htdocs\etic-system\system\Common.php(909): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#8 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(1545): session()
#9 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarReporteBaseLine()
#10 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#11 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-11-03 13:06:14 --> Uncaught ErrorException: Cannot modify header information - headers already sent by (output started at C:\xampp\htdocs\etic-system\app\Config\Events.php:32) in C:\xampp\htdocs\etic-system\system\Debug\Exceptions.php:107
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'Cannot modify h...', 'C:\\xampp\\htdocs...', 107)
#1 C:\xampp\htdocs\etic-system\system\Debug\Exceptions.php(107): header('HTTP/1.1 500 In...', true, 500)
#2 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#3 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-11-03 13:08:44 --> ini_set(): Session ini settings cannot be changed after headers have already been sent
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'ini_set(): Sess...', 'C:\\xampp\\htdocs...', 71)
#1 C:\xampp\htdocs\etic-system\system\Session\Handlers\FileHandler.php(71): ini_set('session.save_pa...', 'C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\Config\Services.php(570): CodeIgniter\Session\Handlers\FileHandler->__construct(Object(Config\App), '127.0.0.1')
#3 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session(Object(Config\App), false)
#4 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#5 C:\xampp\htdocs\etic-system\system\Config\Services.php(563): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#6 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session()
#7 C:\xampp\htdocs\etic-system\system\Common.php(909): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#8 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(1545): session()
#9 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarReporteBaseLine()
#10 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#11 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-11-03 13:08:44 --> Uncaught ErrorException: Cannot modify header information - headers already sent by (output started at C:\xampp\htdocs\etic-system\app\Config\Events.php:32) in C:\xampp\htdocs\etic-system\system\Debug\Exceptions.php:107
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'Cannot modify h...', 'C:\\xampp\\htdocs...', 107)
#1 C:\xampp\htdocs\etic-system\system\Debug\Exceptions.php(107): header('HTTP/1.1 500 In...', true, 500)
#2 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#3 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-11-03 13:09:54 --> ini_set(): Session ini settings cannot be changed after headers have already been sent
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'ini_set(): Sess...', 'C:\\xampp\\htdocs...', 71)
#1 C:\xampp\htdocs\etic-system\system\Session\Handlers\FileHandler.php(71): ini_set('session.save_pa...', 'C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\Config\Services.php(570): CodeIgniter\Session\Handlers\FileHandler->__construct(Object(Config\App), '127.0.0.1')
#3 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session(Object(Config\App), false)
#4 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#5 C:\xampp\htdocs\etic-system\system\Config\Services.php(563): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#6 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session()
#7 C:\xampp\htdocs\etic-system\system\Common.php(909): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#8 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(1545): session()
#9 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarReporteBaseLine()
#10 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#11 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-11-03 13:09:54 --> Uncaught ErrorException: Cannot modify header information - headers already sent by (output started at C:\xampp\htdocs\etic-system\app\Config\Events.php:32) in C:\xampp\htdocs\etic-system\system\Debug\Exceptions.php:107
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'Cannot modify h...', 'C:\\xampp\\htdocs...', 107)
#1 C:\xampp\htdocs\etic-system\system\Debug\Exceptions.php(107): header('HTTP/1.1 500 In...', true, 500)
#2 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#3 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-11-03 13:10:19 --> ini_set(): Session ini settings cannot be changed after headers have already been sent
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'ini_set(): Sess...', 'C:\\xampp\\htdocs...', 71)
#1 C:\xampp\htdocs\etic-system\system\Session\Handlers\FileHandler.php(71): ini_set('session.save_pa...', 'C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\Config\Services.php(570): CodeIgniter\Session\Handlers\FileHandler->__construct(Object(Config\App), '127.0.0.1')
#3 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session(Object(Config\App), false)
#4 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#5 C:\xampp\htdocs\etic-system\system\Config\Services.php(563): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#6 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session()
#7 C:\xampp\htdocs\etic-system\system\Common.php(909): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#8 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(1545): session()
#9 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarReporteBaseLine()
#10 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#11 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-11-03 13:10:19 --> Uncaught ErrorException: Cannot modify header information - headers already sent by (output started at C:\xampp\htdocs\etic-system\app\Config\Events.php:32) in C:\xampp\htdocs\etic-system\system\Debug\Exceptions.php:107
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'Cannot modify h...', 'C:\\xampp\\htdocs...', 107)
#1 C:\xampp\htdocs\etic-system\system\Debug\Exceptions.php(107): header('HTTP/1.1 500 In...', true, 500)
#2 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#3 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-11-03 13:10:43 --> ini_set(): Session ini settings cannot be changed after headers have already been sent
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'ini_set(): Sess...', 'C:\\xampp\\htdocs...', 71)
#1 C:\xampp\htdocs\etic-system\system\Session\Handlers\FileHandler.php(71): ini_set('session.save_pa...', 'C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\Config\Services.php(570): CodeIgniter\Session\Handlers\FileHandler->__construct(Object(Config\App), '127.0.0.1')
#3 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session(Object(Config\App), false)
#4 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#5 C:\xampp\htdocs\etic-system\system\Config\Services.php(563): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#6 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session()
#7 C:\xampp\htdocs\etic-system\system\Common.php(909): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#8 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(1545): session()
#9 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarReporteBaseLine()
#10 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#11 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-11-03 13:10:43 --> Uncaught ErrorException: Cannot modify header information - headers already sent by (output started at C:\xampp\htdocs\etic-system\app\Config\Events.php:32) in C:\xampp\htdocs\etic-system\system\Debug\Exceptions.php:107
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'Cannot modify h...', 'C:\\xampp\\htdocs...', 107)
#1 C:\xampp\htdocs\etic-system\system\Debug\Exceptions.php(107): header('HTTP/1.1 500 In...', true, 500)
#2 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#3 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-11-03 13:12:55 --> ini_set(): Session ini settings cannot be changed after headers have already been sent
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'ini_set(): Sess...', 'C:\\xampp\\htdocs...', 71)
#1 C:\xampp\htdocs\etic-system\system\Session\Handlers\FileHandler.php(71): ini_set('session.save_pa...', 'C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\Config\Services.php(570): CodeIgniter\Session\Handlers\FileHandler->__construct(Object(Config\App), '127.0.0.1')
#3 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session(Object(Config\App), false)
#4 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#5 C:\xampp\htdocs\etic-system\system\Config\Services.php(563): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#6 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session()
#7 C:\xampp\htdocs\etic-system\system\Common.php(909): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#8 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(1545): session()
#9 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarReporteBaseLine()
#10 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#11 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-11-03 13:12:55 --> Uncaught ErrorException: Cannot modify header information - headers already sent by (output started at C:\xampp\htdocs\etic-system\app\Config\Events.php:32) in C:\xampp\htdocs\etic-system\system\Debug\Exceptions.php:107
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'Cannot modify h...', 'C:\\xampp\\htdocs...', 107)
#1 C:\xampp\htdocs\etic-system\system\Debug\Exceptions.php(107): header('HTTP/1.1 500 In...', true, 500)
#2 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#3 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-11-03 13:13:06 --> ini_set(): Session ini settings cannot be changed after headers have already been sent
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'ini_set(): Sess...', 'C:\\xampp\\htdocs...', 71)
#1 C:\xampp\htdocs\etic-system\system\Session\Handlers\FileHandler.php(71): ini_set('session.save_pa...', 'C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\Config\Services.php(570): CodeIgniter\Session\Handlers\FileHandler->__construct(Object(Config\App), '127.0.0.1')
#3 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session(Object(Config\App), false)
#4 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#5 C:\xampp\htdocs\etic-system\system\Config\Services.php(563): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#6 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session()
#7 C:\xampp\htdocs\etic-system\system\Common.php(909): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#8 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(1545): session()
#9 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarReporteBaseLine()
#10 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#11 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-11-03 13:13:06 --> Uncaught ErrorException: Cannot modify header information - headers already sent by (output started at C:\xampp\htdocs\etic-system\app\Config\Events.php:32) in C:\xampp\htdocs\etic-system\system\Debug\Exceptions.php:107
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'Cannot modify h...', 'C:\\xampp\\htdocs...', 107)
#1 C:\xampp\htdocs\etic-system\system\Debug\Exceptions.php(107): header('HTTP/1.1 500 In...', true, 500)
#2 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#3 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-11-03 13:17:04 --> ini_set(): Session ini settings cannot be changed after headers have already been sent
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'ini_set(): Sess...', 'C:\\xampp\\htdocs...', 71)
#1 C:\xampp\htdocs\etic-system\system\Session\Handlers\FileHandler.php(71): ini_set('session.save_pa...', 'C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\Config\Services.php(570): CodeIgniter\Session\Handlers\FileHandler->__construct(Object(Config\App), '127.0.0.1')
#3 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session(Object(Config\App), false)
#4 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#5 C:\xampp\htdocs\etic-system\system\Config\Services.php(563): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#6 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session()
#7 C:\xampp\htdocs\etic-system\system\Common.php(909): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#8 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(1545): session()
#9 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarReporteBaseLine()
#10 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#11 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-11-03 13:17:04 --> Uncaught ErrorException: Cannot modify header information - headers already sent by (output started at C:\xampp\htdocs\etic-system\app\Config\Events.php:32) in C:\xampp\htdocs\etic-system\system\Debug\Exceptions.php:107
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'Cannot modify h...', 'C:\\xampp\\htdocs...', 107)
#1 C:\xampp\htdocs\etic-system\system\Debug\Exceptions.php(107): header('HTTP/1.1 500 In...', true, 500)
#2 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#3 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-11-03 13:17:16 --> count(): Argument #1 ($value) must be of type Countable|array, null given
#0 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(1557): count(NULL)
#1 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarReporteBaseLine()
#2 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#3 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2023-11-03 13:18:14 --> ini_set(): Session ini settings cannot be changed after headers have already been sent
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'ini_set(): Sess...', 'C:\\xampp\\htdocs...', 71)
#1 C:\xampp\htdocs\etic-system\system\Session\Handlers\FileHandler.php(71): ini_set('session.save_pa...', 'C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\etic-system\system\Config\Services.php(570): CodeIgniter\Session\Handlers\FileHandler->__construct(Object(Config\App), '127.0.0.1')
#3 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session(Object(Config\App), false)
#4 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(189): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#5 C:\xampp\htdocs\etic-system\system\Config\Services.php(563): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#6 C:\xampp\htdocs\etic-system\system\Config\BaseService.php(248): CodeIgniter\Config\Services::session()
#7 C:\xampp\htdocs\etic-system\system\Common.php(909): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#8 C:\xampp\htdocs\etic-system\app\Controllers\Inventarios.php(1545): session()
#9 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Inventarios->generarReporteBaseLine()
#10 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Inventarios))
#11 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-11-03 13:18:14 --> Uncaught ErrorException: Cannot modify header information - headers already sent by (output started at C:\xampp\htdocs\etic-system\app\Config\Events.php:32) in C:\xampp\htdocs\etic-system\system\Debug\Exceptions.php:107
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'Cannot modify h...', 'C:\\xampp\\htdocs...', 107)
#1 C:\xampp\htdocs\etic-system\system\Debug\Exceptions.php(107): header('HTTP/1.1 500 In...', true, 500)
#2 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#3 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
ERROR - 2023-11-03 13:30:49 --> Error connecting to the database: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
CRITICAL - 2023-11-03 13:30:49 --> Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
#0 C:\xampp\htdocs\etic-system\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\etic-system\system\Database\BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT `Id_Usua...', Array, false)
#2 C:\xampp\htdocs\etic-system\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\etic-system\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\etic-system\app\Models\UsuariosMdl.php(64): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\etic-system\app\Controllers\Login.php(33): App\Models\UsuariosMdl->obtenerUsuario(Array)
#6 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(824): App\Controllers\Login->acceso()
#7 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\xampp\htdocs\etic-system\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\etic-system\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
